openvdb_points/tools/AttributeSet.h

```diff
@@ -305,6 +337,7 @@ class AttributeSet::Descriptor
     size_t insert(const std::string& name, const NamePair& typeName);
     NameToPosMap                mNameMap;
     std::vector<NamePair>       mTypes;
+    NameToPosMap                mGroupMap;
```

Comment:
Yep. "... current understanding is that there is one *bit* per point". (think that was a typo). Also the point about the offset being relative to the group attributes only is important, otherwise sounds correct.